FreeCAD/FreeCAD
# [TD]Expose DimExtent in Python

Solves issue #13728
The makeExtentDim() function now returns an object to Python, which properties can be used.

Macro to test:

```
'''
- Select an existing view
- Start the macro

A horizontal Extent Dimension is created.
The view's scale, and the extent dimension's raw value is printed.
'''
import TechDraw
import TechDrawTools.TDToolsUtil as Utils
view = Utils.getSelView()
scale = view.getScale()
extentDim = TechDraw.makeExtentDim(view,[],0)
value = extentDim.getRawValue()
print(scale,value)
view.requestPaint()
```

@WandererFan:
Commenting of the three lines
    //if (!dvp) {
    //    return;
    //}
in function makeExtentDim() may be a weak solution. Please check.
